samples/zigbee/ncp/prj_usb.conf

```diff
@@ -4,19 +4,15 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-# Configure NCP sample to use USB CDC ACM 0 serial for communication with Host
-# and inform application that CDC ACM requires the line control signals to be configured.
-CONFIG_ZIGBEE_UART_DEVICE_NAME="CDC_ACM_0"
+# Inform application that CDC ACM requires the line control signals to be configured.
 CONFIG_ZIGBEE_UART_SUPPORTS_FLOW_CONTROL=y
 
 # Configure NCP sample to print ZBOSS stack logs in binary format using "UART_1" device
 CONFIG_ZBOSS_TRACE_BINARY_LOGGING=y
-CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME="UART_1"
```

Comment:
I've found an issue here. `nrf52840dongle_nrf52840` doesn't have `UART_1`. It's now showing up because devices are resolved at compile time. What should we do here?

Reply:
@tomchy @sebastiandraus ping

Reply:
The UART peripheral should be there, it is just not exposed through segger CDC class.
We may change it to use the native CDC class (see `Use this configuration to print ZBOSS stack logs in binary format using USB "CDC_ACM_1" device` section). @sebastiandraus : Do you remember if there were any issues with using this configuration as the default one?

Reply:
The problem is that `nrf52840dongle_nrf52840` doesn't enable `uart1`, not sure how it could work before. `device_get_binding("UART_1") == NULL` for that board.

Reply:
@tomchy I don't remember any issues with that - I think we can enable zboss traces over second instance of USB CDC ACM.
@gmarull Since `ncp` sample is only built for `nrf52840dongle_nrf52840` with `prj_usb.conf`, can overlays for it be simplified (remove basic overlay, leave only `_usb`, possibly remove `_common dtsi`)?

Reply:
> @tomchy I don't remember any issues with that - I think we can enable zboss traces over second instance of USB CDC ACM. 

Should be possible, but current board overlays don't seem to define a 2nd CDC, right?

> @gmarull Since `ncp` sample is only built for `nrf52840dongle_nrf52840` with `prj_usb.conf`, can overlays for it be simplified (remove basic overlay, leave only `_usb`, possibly remove `_common dtsi`)?

Yes, I'll do that then.

Reply:
Please see my latest changeset

Reply:
> Looks fine but `CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING` and `CONFIG_USB_COMPOSITE_DEVICE` would be missing in case of `_dongle`. Maybe enabling zboss traces is not the way. Would this be possible not to set `ncs,zboss-trace-uart` somehow?

Is code prepared to handle a non-existing device?

Reply:
If we can get passed the ` static const struct device *uart_dev = DEVICE_DT_GET(DT_CHOSEN(ncs_zboss_trace_uart));`, the code should handle that.
The `DEVICE_DT_GET` will cause compilation error in such case, can we somehow avoid that?

Reply:
If the implementation is prepared to make `ncs,zboss-trace-uart` optional, there's `DEVICE_DT_GET_OR_NULL`.